File: 101-109/how_old_is_teddy.rb
=begin

Build a program that randomly generates and prints Teddy's age. To get the age,
you should generate a random number between 20 and 200.

Example ouput:
=> Teddy is 69 years old!

=end

age = rand(20..200)
puts  "Teddy is #{age} years old!"

=begin

Modify this program to ask for a name, and then print the age for that person.
For an extra challenge, use "Teddy" as the name if no name is entered.

=end


puts "What's your name?"
name = gets.chomp
age = rand(20..200)

puts "#{name == "" ? "Teddy" : name} is #{age} years old!"

File: javascript/small_problems/how_old_is_teddy.js
// Build a program that randomly generates and logs Teddy's age to the console.
// Have the age be a random number between 20 and 200 (inclusive).
//
// Example Output
//
// Teddy is 69 years old!

function randonBetween(min, max) {
  return Math.floor(Math.random() * (max - min + 1)) + min;
}

var age = randonBetween(20, 200);

console.log('Teddy is ' + age + ' years old!');
